MaxText/layers/attentions.py

```diff
@@ -1707,24 +2019,25 @@ def apply_rotary_embedding(self, inputs: Array, name: str, inputs_positions: Opt
     rope_type = self.config.rope_type.lower()
     rope_use_scale = self.config.rope_use_scale
     if self.is_vision:
-      rotary_embedding = llama_vision_rotary_embedding_as_linen(
+      rotary_embedding = LlamaVisionRotaryEmbedding(
```

Comment:
Looks a little bit weird that these are not assigned to an attribute. Can we move them to `__init__`?